libp2p/protocols/kademlia.nim

```diff
@@ -48,14 +48,22 @@ proc bootstrap*(
       kad.switch.peerStore[AddressBook][p] = peer.addrs
 
     # bootstrap node replied succesfully. Adding to routing table
-    discard kad.rtable.insert(b.peerId)
+    discard kad.rtable.insert(peerId)
 
   let key = PeerId.random(kad.rng).valueOr:
     doAssert(false, "this should never happen")
     return
   discard await kad.findNode(key.toKey())
   info "Bootstrap lookup complete"
 
+proc bootstrap*(
+    kad: KadDHT, bootstrapNodes: seq[PeerInfo]
+) {.async: (raises: [CancelledError]).} =
+  var nodes: seq[(PeerId, seq[MultiAddress])]
+  for node in bootstrapNodes:
+    nodes.add((node.peerId, node.addrs))
```

Comment:
Could mapIt be used for this? i.e. `let nodes = bootstrapNodes.mapIt((it.peerId, it.addrs))`